transform/publication_indexer_test.go

```diff
@@ -48,5 +49,5 @@ func TestPublicationResourceToDoc(t *testing.T) {
 	assert.Equal(t, abstract, doc.Get("abstract_tesim"))
 	assert.Equal(t, publisher, doc.Get("publisher_ssim"))
 	assert.Equal(t, description, doc.Get("description_tesim"))
-
+	assert.Equal(t, 2004, doc.Get("created_year_isim"))
```

Comment:
What do you feel about the `_isim` suffix here? @jcoyne

Reply:
Ping @jcoyne

Reply:
LGTM